Assets/Scripts/UI/PlayerHUDController.cs

```diff
@@ -55,12 +56,26 @@ public class PlayerHUDController : MonoBehaviour
     [SerializeField]
     private PopupSpammer popupSpammer;
     public PopupSpammer PopupSpammer => popupSpammer;
+    [SerializeField]
+    private Image speedLines;
+    [SerializeField]
+    private AnimationCurve speedLineEase;
+    private Material speedLinesMaterial;
+    private float oldLargeVelocity = 0f;
+    private const float lineDampeningVelocity = 11f;
+    private const float lineRemovalMultiplier = 0.8f;
+    private const float lineVelocityDampeningX = 0.25f;
+    private const float lineVelocityDampeningY = 0.1f;
```

Comment:
These parameters could be serialized and documented with tooltips

Reply:
If I serialize them they can't be const. I'll add devcomments instead